tests/app/wsgi.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
This project is usable in a wsgi application ?

Reply:
it can "work" but the socket feature won't work

Reply:
Is the application usable without the socket ? If the answer is no, then you should remove this file and add in the readme that this application works only in a ASGI project.